exec/java-exec/src/test/java/org/apache/drill/exec/rpc/user/TemporaryTablesAutomaticDropTest.java

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
Maybe add a test for the mkdirs conditions discussed above? Make sure that permissions are set on the entire set of newly created folders?